_layouts/post.html

```diff
@@ -81,14 +81,14 @@ <h5 class="index-headline featured"><span>Written by</span></h5>
               <p class="published">Published <time datetime="{{ page.date | date: '%F %R' }}">{{ page.date | date_to_string }}</time></p>
             </section>
           </div>
-          {{/post}}
+          {{post}}
```

Comment:
It was a warning reported by Jekyll and it looks like an accidental typo to me...